akka-stream-tests/src/test/scala/akka/stream/scaladsl/GraphConcatSpec.scala

```diff
@@ -97,6 +104,7 @@ class GraphConcatSpec extends TwoStreamsSetup {
     }
 
     "work with one delayed failed and first nonempty publisher" in {
+      pending // FIXME: This relies on materialization order!!
```

Comment:
this worked before because of the particular order we materialized things. We should revisit this later.